Cargo.toml

```diff
@@ -9,6 +9,7 @@ license = "Apache-2.0 AND BSD-3-Clause"
 default = ["elf"]
 elf = []
 bzimage = []
+arm64_pe = []
```

Comment:
Please add a TODO & an issue so we can add target-dependent features when https://github.com/rust-lang/cargo/issues/1197 makes it into `stable`. I'd like to see the `arm64_pe` only available on `aarch64` (and maybe just called `pe`).

Reply:
Before target-dependent feature come to stable, is there a workaround in our CI? For arm build and test, now we need to enable `arm64_pe` but disable `elf`. 
In local test, I do with command `cargo test --features arm64_pe --no-default-features`.

Reply:
Will create an issue to track it after merging.